packages/cli/src/commands/init/templates/app-react-email/ts/app/novu/emails/novu-onboarding-email.tsx

```diff
@@ -64,22 +66,22 @@ export const NovuWelcomeEmail = ({
               return (
                 <Section key={componentIndex}>
                   {component.type === "heading" ? (
-                    <Column>
-                      <h1 className={`text-${component.align}`}>
+                    <Section>
+                      <Heading as="h1" className={`text-${component.align}`}>
                         {component.text}
-                      </h1>
-                    </Column>
+                      </Heading>
+                    </Section>
                   ) : null}
 
                   {component.type === "button" ? (
-                    <Column className={`text-${component.align}`}>
+                    <Section className={`text-${component.align}`}>
                       <Button
                         href={"http://localhost:2022"}
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** ‏ I've clicked on the button while looking at the editor and it causes a full-page reload / navigation away from where the user is... Would a "#" perhaps be better? Or even a link to our docs so that it showcases going elsewhere both in the preview and in the received test email?